MediaManager/Playback/IPlaybackManager.cs

```diff
@@ -85,7 +85,7 @@ public interface IPlaybackManager : INotifyPropertyChanged
         /// <summary>
         /// Toggles between the different repeat: modes None, RepeatOne and RepeatAll
         /// </summary>
-        void ToggleRepeat();
+        void SetRepeatMode(RepeatMode mode);
```

Comment:
I would make RepeatMode a property when changing it like this. Also add an extension method or something to Toggle, because that was used quite often in the past.